src/components/VideoChatMenu.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Popover from './Popover';
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+import MenuItem from './MenuItem';
+import openURLInNewTab from '../libs/openURLInNewTab';
+import ZoomIcon from '../../assets/images/Zoom - Blue.svg';
+import GoogleMeetIcon from '../../assets/images/google-meet.svg';
+import CONST from '../CONST';
+
+const propTypes = {
+    // State that determines whether to display the create menu or not
+    isVisible: PropTypes.bool.isRequired,
+
+    // Callback that determines behavior when menu is closed
+    onClose: PropTypes.func.isRequired,
+
+    ...windowDimensionsPropTypes,
+};
+
+class VideoChatMenu extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const menuItemData = [
+            {
+                icon: ZoomIcon,
+                text: 'Zoom',
+                onPress: () => openURLInNewTab(CONST.NEW_ZOOM_MEETING_URL),
+            },
+            {
+                icon: GoogleMeetIcon,
+                text: 'Google Meet',
+                onPress: () => openURLInNewTab(CONST.NEW_GOOGLE_MEET_MEETING_URL),
+            },
+        ].map(item => ({
+            ...item,
+            onPress: () => {
+                this.props.onItemSelected();
```

Comment:
I don't see an `onItemSelected` prop, only `onClose`

Reply:
I think you might want:

```
item.onPress();
this.props.onClose();
```

So that clicking any of the menu items closes the Popover